CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+Contributing to ceph-ansible
+==============================
+
+1. Follow the [commit guidelines](#commit-guidelines)
+
+
+Commit guidelines
+-----------------
+- All commits should have a subject and a body
```

Comment:
Is this going to be forcefully checked regardless? Sounds a bit extreme to require a body for every single commit

Reply:
This is not yet forcefully check but will surely.
That is not extreme when considering supporting the software.
When you have to debug, a commit message is always useful to understand why that change was made. A single line patch can lead to a very long commit message.

So yes, a detailed commit message is mandatory.